### ISSUE TITLE
Update Database error page for 5.0

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#19718]
```csharp
public void ConfigureServices(IServiceCollection services)
{
    services.AddDbContext<ApplicationDbContext>(options =>
        options.UseSqlServer(
            Configuration.GetConnectionString("DefaultConnection")));
    services.AddDatabaseDeveloperPageExceptionFilter();  // <------------------------------
    services.AddDefaultIdentity<IdentityUser>(options => options.SignIn.RequireConfirmedAccount = true)
        .AddEntityFrameworkStores<ApplicationDbContext>();
    services.AddRazorPages();
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0e03fdf4-b19b-17c3-04f4-336ab6d2ee06
* Version Independent ID: 38515dfb-91a5-b395-db9d-084bbaf095c8
* Content: [Handle errors in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-3.1)
* Content Source: [aspnetcore/fundamentals/error-handling.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/error-handling.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by serpent5** [comment]
@Rick-Anderson Is it alright if I take this one?

**Comment by Rick-Anderson** [comment]
Yes, please do.

**Comment by Rick-Anderson** [comment]
@serpent5  probably won't do whole file monikers. Just put monikers around things that  change. If you need more than 3 moniker sections each, it's time to go to whole file monikers.

```
::: moniker range=">= aspnetcore-5.0"

::: moniker-end

::: moniker range="< aspnetcore-5.0"

::: moniker-end

::: moniker range="< aspnetcore-3.0"

```

**Comment by serpent5** [comment]
@Rick-Anderson It looks like you fixed this one in #19741.

**Comment by Rick-Anderson** [comment]
Yup, sorry for the false alarm.
